pkg/model/manifest.go

```diff
@@ -636,8 +640,18 @@ func Read(bytes []byte) (*Manifest, error) {
 	}
 
 	hasShownWarning := false
-	for _, d := range manifest.Dev {
-		if (d.Image.Context != "" || d.Image.Dockerfile != "") && !hasShownWarning {
+	for devKey, dev := range manifest.Dev {
+		if shouldBeSanitized(devKey) {
```

Comment:
We should probably check if there is a `global forward`/`build` with the same unsanitised key